lib/integration/dbConnector.ts

```diff
@@ -0,0 +1,10 @@
+import { Client } from "pg";
+
+const dbConnector = (connexion?: string): Client => {
```

Comment:
Not sure again why this is intentionally misspelled. Could just be called connection and it wouldn't break anything